documentation/plugin/customisation.rst

```diff
@@ -221,6 +221,25 @@ Then, overwrite the ``/favicon.ico`` route which FlexMeasures uses to get the fa
 Here we assume your favicon is a PNG file. You can also use a classic `.ico` file, then your mime type probably works best as ``image/x-icon``.
 
 
+Customizing the breadcrumbs
+---------------------------------
+
+On asset and sensor pages, we show breadcrumbs on top (e.g. Account -> Asset -> ChildAsset -> Sensor).
+Say you want to adapt this, so that some asset has a unique breadcrumb path
+
+Add an attribute to an asset or sensor "breadcrumb_ancestry", e.g.:
+
+.. code-block:: python
+
+    my_asset.attributes["breadcrumb_ancestry"] = 
+        [
+            {"url": my_url, "name": "Top-level", "type": "Asset"},
+            {"url": another_url, "name": "2nd-level", "type": "Asset"}
```

Comment:
Why is there a need to define the type? The docs could mention what I can do with this as a plugin developer.

Reply:
I actually can't find code that makes use of that. Maybe it could be used to display them differently.

That reminds me, I can make the breadcrumb HTML part of its own Jinja block. Then plugins could change the design if they wanted.